docs/architecture/psa-migration/psa-limitations.md

```diff
@@ -142,7 +158,8 @@ match a limitation of the PSA API.
 It is unclear what parameters people use in practice. It looks like by default
 OpenSSL picks saltlen = keylen - hashlen - 2 (tested with openssl 1.1.1f).
 The `certool` command provided by GnuTLS seems to be picking saltlen = hashlen
-by default (tested with GnuTLS 3.6.13).
+by default (tested with GnuTLS 3.6.13). FIPS 186-4 recommends saltlen >=
+hashlen.
```

Comment:
Sorry, I made a typo in my comment. It's the other way round: FIPS 186 (every version) requires 0 ≤ slen ≤ hlen. That is, the salt must not be longer than the hash, and may be shorter, even empty.

Reply:
Aw, I should have checked. Thanks for the correction.